core-devel/binutils/autobuild/patches/0005-bfd-pic.patch

```diff
@@ -0,0 +1,29 @@
+Author: Balint Reczey <balint@balintreczey.hu>
+Description: Build libbfd with -fPIC to allow linking with PIE binaries
+
+Index: b/bfd/Makefile.am
+===================================================================
+--- a/bfd/Makefile.am
++++ b/bfd/Makefile.am
+@@ -51,7 +51,7 @@ ZLIBINC = @zlibinc@
+ 
+ WARN_CFLAGS = @WARN_CFLAGS@
+ NO_WERROR = @NO_WERROR@
+-AM_CFLAGS = $(WARN_CFLAGS) $(ZLIBINC)
++AM_CFLAGS = $(WARN_CFLAGS) $(ZLIBINC) -fPIC
```

Comment:
Well, I'm concerning this patch may have effect on other architecture. It's grabbed from Debian and probably necessary for MIPS64.

Reply:
Then you should consider adding a `.mips64el` suffix or something that matches the architecture name - making it `0005-bfd-pic.patch.mips64el`.